lib/data/network/exception/dio_duplicate_request_exception.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:dio/dio.dart';
+import 'package:equatable/equatable.dart';
+
+class DioDuplicateRequestException extends DioException with EquatableMixin {
+  DioDuplicateRequestException({required RequestOptions requestOptions})
```

Comment:
why not `DioDuplicateDownloadException`?

Reply:
done